cdc/sink/dmlsink/mq/dispatcher/partition/dispatcher.go

```diff
@@ -21,5 +21,5 @@ import (
 type Dispatcher interface {
 	// DispatchRowChangedEvent returns an index of partitions according to RowChangedEvent.
 	// Concurrency Note: This method is thread-safe.
-	DispatchRowChangedEvent(row *model.RowChangedEvent, partitionNum int32) int32
+	DispatchRowChangedEvent(row *model.RowChangedEvent, partitionNum int32) (int32, *string)
```

Comment:
update the function comment

Reply:
What is the usage of the second returned value, it's set as `num2str(sum32)`